daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala

```diff
@@ -357,8 +360,8 @@ class OrderingSpec extends WordSpec with Matchers with TableDrivenPropertyChecks
       STextMap(HashMap.empty) ->
         STextMap(HashMap("a" -> lfFunction)),
       SGenMap.Empty -> SGenMap(SInt64(0) -> lfFunction),
-      SVariant(VariantTypeCon, VariantCon1, SInt64(0)) ->
-        SVariant(VariantTypeCon, VariantCon2, lfFunction),
+      SVariant(VariantTypeCon, VariantCon1, 1, SInt64(0)) ->
```

Comment:
Why does this constructor rank not start with `0`?

Reply:
I will fix that.